store.go

```diff
@@ -712,20 +714,21 @@ func GetStore(options types.StoreOptions) (Store, error) {
 		autoNsMaxSize = AutoUserNsMaxSize
 	}
 	s := &store{
-		runRoot:         options.RunRoot,
-		graphLock:       graphLock,
-		graphRoot:       options.GraphRoot,
+		additionalGIDs:  nil,
+		additionalUIDs:  nil,
+		autoNsMaxSize:   autoNsMaxSize,
+		autoNsMinSize:   autoNsMinSize,
+		autoUsernsUser:  options.RootAutoNsUser,
+		disableVolatile: options.DisableVolatile,
+		gidMap:          copyIDMap(options.GIDMap),
 		graphDriverName: options.GraphDriverName,
+		graphLock:       graphLock,
 		graphOptions:    options.GraphDriverOptions,
+		graphRoot:       options.GraphRoot,
+		pullOptions:     options.PullOptions,
+		runRoot:         options.RunRoot,
 		uidMap:          copyIDMap(options.UIDMap),
-		gidMap:          copyIDMap(options.GIDMap),
-		autoUsernsUser:  options.RootAutoNsUser,
-		autoNsMinSize:   autoNsMinSize,
-		autoNsMaxSize:   autoNsMaxSize,
-		additionalUIDs:  nil,
-		additionalGIDs:  nil,
 		usernsLock:      usernsLock,
-		disableVolatile: options.DisableVolatile,
```

Comment:
This makes it harder to use `git annotate` to find which patch/PR introduced a particular option, which hopefully includes rationale for the changes.